### ISSUE TITLE
After connection stablished Navigated to http://localhost:8080/ 

### ISSUE THREAD

**Issue by imtiazwazir** [hapijs/nes#170]
After connection stablished its reloading page twice on client side in result can't get results from server side.

console output without preserve log enabled:
<img width="1199" alt="screen shot 2016-12-27 at 9 13 40 pm" src="https://cloud.githubusercontent.com/assets/1964969/21503442/6971793c-cc79-11e6-99ac-99420321b9b4.png">

console output with preserve log enabled:
<img width="1202" alt="screen shot 2016-12-27 at 9 14 35 pm" src="https://cloud.githubusercontent.com/assets/1964969/21503462/7fec5af6-cc79-11e6-9eb1-54f3ce808982.png">

subscription and publish code (server side):
`
  server.subscription('/slides/{clientName}');

  server.route({
    method: 'GET',
    path: '/',
    config: {
      description: 'Return the home page',
      handler: (request, reply) => {
        server.publish('/slides/about', {file: 'file-name'});
        return reply.view('home', {home: true, title: 'Home'});
      }
    }
  });
`

Client side code:

`
client.connect((err) => {
  if (err) {
    console.error(err.message)
    process.exit(1)
  }

  console.log('connected')

  client.subscribe('/slides/about', (message) => {
    console.log(message)
  }, (err) => {
    if (err) {
      console.error(err.message)
      process.exit(1)
    }

    console.log('subscribed')
  })
})
`

Any idea what is going on and why page reload twice?

**Comment by imtiazwazir** [comment]
following is by plugin's code

`'use strict';

exports.register = (server, options, next) => {
  server.register([{
    register: require('nes'),
    options: {
      heartbeat: {
          interval: 30000,
          timeout: 10000
      }
    }
  }], (err) => {});

  return next();
}

exports.register.attributes = {
  name: 'hapi-nes'
}`

let me know if you require anything

**Comment by hueniverse** [comment]
Is this all the code or is there more on the client side? There is no way for me to know what is going on here. I also don't have the time to help you add more debugging but you need to log every event on the client, especially errors and disconnects and see if that gives you a clue.

**Comment by imtiazwazir** [comment]
No, there isn't any event on client side still I am attaching my code below.

<img width="813" alt="screen shot 2016-12-28 at 10 24 47 pm" src="https://cloud.githubusercontent.com/assets/1964969/21531108/51b5e636-cd67-11e6-9752-c1a8cbd8478c.png">

<img width="850" alt="screen shot 2016-12-29 at 1 35 14 am" src="https://cloud.githubusercontent.com/assets/1964969/21531109/556ab2e8-cd67-11e6-8589-faa2018f459f.png">

<img width="1326" alt="screen shot 2016-12-29 at 1 35 22 am" src="https://cloud.githubusercontent.com/assets/1964969/21531112/5733abd4-cd67-11e6-9a2a-45284ad3adbb.png">

I have tried broadcast message and its working fine only Pub/Sub thingie is not working

**Comment by imtiazwazir** [comment]
Its fixed now looks like its working fine on live server, node-dev plugin was giving some issues

**Comment by imtiazwazir** [comment]
Thanks for you help :)
